ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-nonprod.json

```diff
@@ -0,0 +1,20 @@
+[
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "NOC_REQUEST",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "caseworker-civil-solicitor"
+        ],
+        "CRUD": "R"
+      },
+      {
+        "UserRoles": [
+          "caseworker-civil-admin"
```

Comment:
i think you can add this under line 8: 
```
"UserRoles": [
          "caseworker-civil-solicitor",
          "caseworker-civil-admin"
        ],

```
since they both have the same CRUD

Reply:
Thanks, I've updated!